__tests__/index.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { render, screen } from '@testing-library/react'
+import Home from '../pages/index'
+import '@testing-library/jest-dom'
```

Comment:
all the import from 3rd party packages must be on top and import closest to the file must be on the bottom